docs/dev/developer-guides/bridging/bridging-asset.md

```diff
@@ -173,7 +171,35 @@ Users must call the `withdraw` method on the L2 bridge contract, which will trig
 - After the method is called, the funds are unlocked from the L1 bridge and sent to the withdrawal recipient.
 
 ::: warning
-
 On the testnet environment, we automatically finalize all withdrawals, i.e., for every withdrawal, we will take care of it by making an L1 transaction that proves the inclusion for each message.
-
 :::
+
+## Custom bridges
+
+Implement your own custom bridge by creating a new `L1ERC20BridgeFactory` and using it to deploy a custom bridge by passing the `CONTRACTS_DIAMOND_PROXY_ADDR` and an `allowlist` for the contract signer to the `deploy` function. 
```

Comment:
I'm not sure it's true. 
`L1ERC20Bridge` and `Factory` is our implementation of the ERC20 bridge. 

And everything you described below is about our bridge implementation, not the custom one, is it on purpose?

Reply:
I wrote this from looking at the example code provided which is where I got the snippet I put below the text. The link to the example was in the ticket here: https://github.com/matter-labs/zksync-2-dev/pull/1633/files#diff-143fd8bd5920c0327f92ee25b6bd377322d122d3d45424a2e5dc92cc60a2096e please go ahead and correct or if it's a bigger job, we can have a quick call?

Reply:
got it, thank you

Reply:
DONE, please check.